### PR TITLE
Limit bookings

### PR DESCRIPTION
**Pull request by ccastro1987**
I implemented this feature request - do you want to check out the code before I push it to staging?

I think there are probably many ways to do this one, but this is what looked the most simple to me.
Have a look and let me know if this is ok or if there is a better way of doing it...

### CODE REVIEW

File: build/Mstc/MSTC.Web/Controllers/EventController.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ public EventController()
 		}
 
 		[HttpGet]
-		public IEnumerable<EventType> BookableEvents(bool futureEventsOnly, bool withSlotsOnly, bool isAdmin = false)
+		public IEnumerable<EventType> BookableEvents(bool futureEventsOnly, bool withSlotsOnly, bool isAdmin = false, int limitBooking = 2)
```
Comment (MikeHook) [inline build/Mstc/MSTC.Web/Controllers/EventController.cs:32]: This method is also called from the event-admin.js script. I don't think you would want to limit the bookings for that query so I think would bebetter to make the limitBooking parameter optional here. It should work OK then as you already made it optional on the repository method :)

File: build/Mstc/MSTC.Web/scripts/mstc/event-booking.js [hunk 2]
```diff
@@ -134,7 +136,7 @@
 	var getEvents = function () {
 		resetProps();
 		$.ajax({
-			url: '/umbraco/api/event/BookableEvents?futureEventsOnly=true&withSlotsOnly=true',
+			url: `/umbraco/api/event/BookableEvents?futureEventsOnly=true&withSlotsOnly=true&limitBooking=${limitBooking}`,
```
Comment (MikeHook) [inline build/Mstc/MSTC.Web/scripts/mstc/event-booking.js:139]: Interesting way to pass a javascript parameter, not thought of doing it like that before!

**Review by MikeHook — CHANGES_REQUESTED** [review]
Looks fine other than the one issue with the admin query.

Thanks, Mike

File: build/Mstc/MSTC.Web/Controllers/EventController.cs (hunk 1 above)
Comment (ccastro1987) [reply build/Mstc/MSTC.Web/Controllers/EventController.cs:32]: agreed!

File: build/Mstc/MSTC.Web/scripts/mstc/event-booking.js (hunk 2 above)
Comment (ccastro1987) [reply build/Mstc/MSTC.Web/scripts/mstc/event-booking.js:139]: 👍

File: build/Mstc/MSTC.Web/Controllers/EventController.cs (hunk 1 above)
Comment (ccastro1987) [reply build/Mstc/MSTC.Web/Controllers/EventController.cs]: I've updated the controller and also removed the limit in the API call

**Pull request merged**